zapisy/apps/defects/templates/showDefect.html

```diff
@@ -0,0 +1,141 @@
+    <h1 class="d-inline-block">
+        {{ defect.name }}
+    </h1>
+    <div class="ml-2 d-flex align-items-center justify-content-end flex-wrap">
+        <div class="d-inline-block mb-3 ml-1">
+            <a class="btn btn-outline-info" href="{% url 'defects:edit_defect' defect.id %}">
+                Edytuj
+            </a>
+            {% if can_delete %}
+            <a
+                    class="btn btn-danger"
+                    href="{% url 'defects:delete_defect' defect.id %}"
+                    onclick="return confirm('Czy na pewno chcesz usunąć tę usterkę?');">
+                Usuń
+            </a>
+            {% endif %}
+        </div>
+    </div>
+</div>
+<table class="table table-bordered table-md-responsive d-print-none" id="table-info">
+    <colgroup>
+        <col class="table-info-type"></col>
+    </colgroup>
+    <tbody>
+    <tr>
+        <th scope="row" style="width: 50%">Miejsce</th>
+        <td>
+            {{ defect.place }}
+        </td>
+    </tr>
+    <tr>
+        <th scope="row">Zgłoszona przez</th>
+        <td>{% if defect.reporter %}
+            {% with defect.reporter.first_name|add:' '|add:defect.reporter.last_name as name %}
+            {{ name }}
+            {% endwith %}
+            {% else %}
+            Brak
+            {% endif %}
+        </td>
+    </tr>
+    <tr>
+        <th scope="row">Stan</th>
+        <td>
+            {{ defect.get_state_display }}
+        </td>
+    </tr>
+    <tr>
+        <th scope="row">Data dodania</th>
+        <td>{{ defect.creation_date }}</td>
+    </tr>
+    <tr>
+        <th scope="row">Data modyfikacji</th>
+        <td>{{ defect.last_modification }}</td>
+    </tr>
+    </tbody>
+</table>
+<div style="word-break: break-word">
+    {% if defect.description != "" %}
+    <h2>Opis usterki</h2>
+    {{ defect.description|linebreaksbr }}
+    {% endif %}
+</div>
+<hr>
+
+<h3>Informacja o zmianach</h3>
+<div id="info-div">
+
+
+    {% if is_defect_manager %}
+    <div id="info-form-div">
+        <form action="{% url 'defects:repair_info' defect.id %}" method="post" id="info-form">
+            {% csrf_token %}
+            {% crispy info_form %}
+            <button class="btn btn-success" type="submit">Edytuj</button>
+        </form>
+    </div>
+    {% else %}
+    <div id="info-text">
+        {% if defect.information_from_defect_manager %}
+        <p>{{ defect.information_from_defect_manager|linebreaksbr }}</p>
+        {% else %}
+        <p>Nie ma jeszcze żadnej informacji o zmianach</p>
+        {% endif %}
+    </div>
+    {% endif %}
+</div>
+
+
+<hr>
+
+<div class="container-fluid">
+    <div class="row">
+        {% for image_url in image_urls %}
+        <div class="col-md-3 pop center-block">
+            <img class="img-thumbnail center-block" src={{ image_url }}>
+        </div>
+        {% endfor %}
+    </div>
+</div>
+
+<div class="modal fade" id="imagemodal" tabindex="-1" role="dialog" a ria-labelledby="myModalLabel"
```

Comment:
```suggestion
<div class="modal fade" id="imagemodal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"
```
Ale oprócz tego wartość tego atrybutu jest dość pozbawiona znaczenia; jeśli nie wymyślimy nic mądrzejszego, to chyba można go w ogóle wywalić.